README.md

```diff
@@ -14,6 +12,10 @@ Improvements to the type system should be discussed on the
 mailing list, although the [issues](https://github.com/python/typing/issues) in this
 repository contain some historic discussions.
 
+For conversations that are more suitable to a chat platform, you can use one of the following:
```

Comment:
Can't comment there but we should remove the mention of Gitter on line 8 above.

Reply:
sharp eye! Done.